### PR TITLE
add full screen plugin for leaflet maps

### PR DESCRIPTION
**Pull request by alexbowen**
no ticket

LeafletJs is a quality library and there is a plethora of functionality and plugins that we could take advantage of. obviously dont want to go overboard but i thought this (as a simple plugin introduction) was quite neat and i feel is of benefit for a mobile/small screen user to use. use the control to toggle default/full screen view and works smoothly cross browser.

these screen shots probably not the best example, would be useful more on busy maps with many markers/polygons

Screenshot 2022-03-30 at 23 56 44

Screenshot 2022-03-30 at 23 57 22

Screenshot 2022-03-30 at 23 57 51

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1234,7 +1234,7 @@
     mkdirp "^1.0.4"
     rimraf "^3.0.2"
 
-"@rails/webpacker@^5.4.3":
+"@rails/webpacker@^5.4.2":
```
Comment (cesidio) [inline yarn.lock:1237]: why this change?

Comment (alexbowen) [reply yarn.lock:1237]: well spotted, i shouldnt have. even tho dont think it matters, ill remove it from this pr

**Review by cesidio — APPROVED** [review]
